idb/postgres/internal/ledger_for_evaluator/ledger_for_evaluator_test.go

```diff
@@ -1,7 +1,8 @@
-package postgres
+package ledgerforevaluator_test
```

Comment:
`package ledger_for_evaluator` to match the directory name?

Reply:
Go (linter) complains about underscores in package names. `_test` ensures that testing is black-box: that we can't e.g. access private struct fields from the tested code.

Reply:
Then maybe it can't live in a directory with underscores in the directory name.

Reply:
Just because go has a strange convention for package names doesn't mean we should propagate it to directory names. Removing underscores from the package name is a good compromise to keep using the linter.